pom.xml

```diff
@@ -44,6 +44,8 @@
   <properties>
     <jenkins.version>2.176.4</jenkins.version>
     <java.level>8</java.level>
+    <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
@timja are you ok to merge it ? Not sure if the findbugs errors are still here

Reply:
I would remove that, given I just removed it in the previous PR

```suggestion
```